index.html

```diff
@@ -362,6 +370,6 @@ <h1 class="Contactme">Contact Me</h1>
         </form>
         
```

Comment:
- [ ] There's an empty space on the right side, I would suggest removing it as this is not part of the design:

![Captura de pantalla 2022-06-01 a la(s) 9 36 05 p  m](https://user-images.githubusercontent.com/83717285/171540317-8bf4082a-a655-4a81-9a23-6f40eb2486ec.png)

Reply:
done